WebDavServer.Infrastructure.FileStorage/Services/PathService.cs

```diff
@@ -77,14 +80,14 @@ public async Task<PathInfo> GetDestinationPathInfoAsync(string relativePath, Can
 
             if (item == null)
             {
-                throw new FileStorageException(ErrorCodes.PartOfPathNotExists);
+                return default;

```

Comment:
use http://www.webdav.org/neon/litmus/ for testing all changes, especially this place